modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc

```diff
@@ -3,49 +3,61 @@
 :parent-context-of-creating-a-workspace-by-importing-the-source-code-of-a-project: {context}
 
 [id="creating-a-workspace-by-importing-the-source-code-of-a-project_{context}"]
-= Creating a workspace by importing the source code of a project
+= Creating a workspace with your project's source code
 
 :context: creating-a-workspace-by-importing-the-source-code-of-a-project
 
-This section describes how to create a new workspace for editing an existing codebase.
+//Titles modified by me without xref changes yet. max-cx
+
+With {prod-short}, you can import and work on your project's current codebase in a workspace.
+
+You can import your project's current codebase into a workspace with any of the following options:
+
+* xref:creating-a-custom-workspace-from-the-dashboard_{context}[Importing a project when creating a workspace]
+* xref:importing-from-the-dashboard-into-an-existing-workspace_{context}[Importing a project into a created workspace]
+* xref:importing-to-a-running-workspace-using-the-git-clone-command_{context}[Importing a project with *Git: Clone* in the GUI]
+* xref:importing-to-a-running-workspace-with-git-clone-in-a-terminal_{context}[Importing a project with `git clone` in a terminal]
 
 .Prerequisites
 
-* A running instance of {prod-short}. To install an instance of {prod-short}, see {link-installing-an-instance}.
+* A running instance of {prod-short}
```

Comment:
Period omitted deliberately. IBM Style Guide: p. 89 "_Write lists so that all or none of the items start with complete sentences. ... If list items comprise only sentence fragments, do not include any end punctuation_."